### PR TITLE
I added IBM

### PR DESCRIPTION
**Pull request by rafiya2003**
## Description

I added IBM at line 6  

**Fixes** #5800

**Fixes** #5800

### CODE REVIEW

File: docs/data-integrations/ibm-db2.mdx [hunk 1]
```diff
@@ -13,12 +13,12 @@ This handler is implemented using `ibm_db/ibm_db_dbi`, a Python library that all
 
 The required arguments to establish a connection are as follows:
 
-* `user` is the username associated with the database.
-* `password` is the password required to authenticate your access to the database.
-* `host` is the hostname or IP address of the server.
-* `port` is the port through which TCP/IP connection is to be made.
-* `database` is the database name to be connected.
-* `schema` is required to get the tables.
+- `user` is the username associated with the database.
```
Comment (martyna-mindsdb) [inline docs/data-integrations/ibm-db2.mdx:16]: You changed `*` into `-`.

Please change it back to `*` for all six points.

**Comment by martyna-mindsdb** [comment]
recheckcla

**Comment by mindsdbadmin** [comment]
**CLA Assistant Lite bot:** <br/>Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/mindsdb/mindsdb_native/blob/stable/assets/contributions-agreement/individual-contributor.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.<br/>
   - - -
   I have read the CLA Document and I hereby sign the CLA
   - - -
   <sub>You can retrigger this bot by commenting **recheck** in this Pull Request</sub>

**Comment by martyna-mindsdb** [comment]
@rafiya2003 
Please see the review comments above. Also, please sign our CLA before we can merge your PR.

**Comment by martyna-mindsdb** [comment]
@rafiya2003 
Before we can merge your PR, please sign our CLA by leaving a comment here.

**Comment by rafiya2003** [comment]
I have read the CLA Document and I hereby sign the CLA

**Pull request merged**